src/bundle/Resources/public/css/modules/field-edit.css

```diff
@@ -0,0 +1,19 @@
+ez-field-edit input:invalid {
```

Comment:
`select` and `textarea` can also be invalid. That said, this kind of rule should also be applied to forms that don't involve fields (ie forms in the admin panel). So I think we should take some time to think about forms in general to not make the same mistake as in PlatformUI where forms were mostly approached for Content edit resulting in poor admin forms.